examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# (c) 2018 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
```

Comment:
It's not needed to update the copyright year, so, don't worry too much about these changes